src/infra/guild-command-manager.ts

```diff
@@ -21,17 +21,32 @@ export class GuildCommandManager {
         try {
             this.finalized = true;
             const rest = new REST({ version: "10" }).setToken(token);
-            M.log("Sending guild commands");
-            await rest.put(
-                Routes.applicationCommands(this.wheatley.id),
-                { body: this.commands },
-            );
-            // Clear any previous guild commands
-            await rest.put(
-                Routes.applicationGuildCommands(this.wheatley.id, TCCPP_ID),
-                { body: [] },
-            );
-            M.log("Finished sending guild commands");
+            if (!this.wheatley.freestanding) {
+                M.log("Sending application commands");
+                await rest.put(
+                    Routes.applicationCommands(this.wheatley.id),
+                    { body: this.commands },
+                );
+                // Clear any previous guild commands
+                await rest.put(
+                    Routes.applicationGuildCommands(this.wheatley.id, this.wheatley.guildId),
+                    { body: [] },
+                );
```

Comment:
This `put(Routes.applicationGuildCommands)` is no longer needed since we never set `applicationGuildCommands`, I think we can clean this section up a lot. It should be ok to put `Routes.applicationCommands` in freestanding mode.

Reply:
I've cleaned up the section, but would prefer to use `applicationGuildCommands` for freestanding. Afaik, application guild commands update faster, so it's more convenient for quick little tests that you would do in freestanding mode.